api/core/execute.go

```diff
@@ -1,32 +1,30 @@
 package core
 
 import (
+	"context"
 	"encoding/json"
 
 	"github.com/mesg-foundation/core/database/services"
-
-	"context"
-
 	"github.com/mesg-foundation/core/execution"
 )
 
-// ExecuteTask will execute a task for a given service
+// ExecuteTask executes a task for a given service.
 func (s *Server) ExecuteTask(ctx context.Context, request *ExecuteTaskRequest) (*ExecuteTaskReply, error) {
 	service, err := services.Get(request.ServiceID)
 	if err != nil {
 		return nil, err
 	}
+
 	var inputs map[string]interface{}
-	err = json.Unmarshal([]byte(request.InputData), &inputs)
-	if err != nil {
+	if err = json.Unmarshal([]byte(request.InputData), &inputs); err != nil {
```

Comment:
usually errs recreated when used within ifs like this `err :=`

Reply:
Done